app/controllers/posts_controller.rb

```diff
@@ -12,4 +12,10 @@ def show
       format.json { render json: @post }
     end
   end
+
+  def increment_likes
+    @post = Post.find(params[:id])
+    @post.increment!(:LikesCounter)
+    redirect_to user_post_path(@post.author_id, @post.id)
+  end
```

Comment:
- [ ] Please we should have the create like method only inside the `LikeController`.
- [ ] Remember we should have only one controller to handle Post actions.